ci-operator/config/stolostron/hypershift-deployment-controller/stolostron-hypershift-deployment-controller-main.yaml

```diff
@@ -24,12 +24,18 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: fast-forward
+- as: backplane-fast-forward
   postsubmit: true
   steps:
     env:
       DESTINATION_BRANCH: backplane-2.1
     workflow: ocm-ci-fastforward
+- as: backplane-z-fast-forward
+  postsubmit: true
+  steps:
+    env:
+      DESTINATION_BRANCH: backplane-2.0
```

Comment:
This will ff main to backplane-2.0, right？But when I tried to merge main to backplane-2.0, there are conflicts, not sure if this can work as expected. https://github.com/stolostron/hypershift-deployment-controller/compare/backplane-2.0...main

Reply:
@zhujian7 thanks for the review.

Yes, we will need https://github.com/stolostron/hypershift-deployment-controller/pull/131 to be merged first for the auto FF to work.